app/Controllers/Persons.php

```
@@ -43,34 +39,88 @@ public function getSuggest(): ResponseInterface
+    {
+        $builder = $this->db->table('attribute_links');
+        $builder->where('person_id', $personId);
+        $builder->where('item_id', null);
+        $builder->where('sale_id', null);
+        $builder->where('receiving_id', null);
+        $builder->where('definition_id', $definitionId);
+
+        return $builder->get()->getRowObject();
+    }
+
+    protected function savePersonAttributes(int $personId, int $definitionFlags): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$definitionFlags'. (undefined)

(UnusedFormalParameter)